build-tools/packages/build-cli/src/base.ts

```diff
@@ -15,11 +15,19 @@ import { rootPathFlag } from "./flags";
 export abstract class BaseCommand extends Command {
     static flags = {
         root: rootPathFlag(),
+        timer: Flags.boolean({
+            default: false,
+            hidden: true
+        }),
         verbose: Flags.boolean({
             char: "v",
             description: "Verbose logging.",
             required: false,
         }),
+        logtime: Flags.boolean({
+            description: "Display the current time on every status message for logging",
+            required: false
+        })
```

Comment:
```suggestion
        logtime: Flags.boolean({
            description: "Display the current time on every status message for logging",
            hidden: true,
            required: false
        })
```

I suggest either hiding this one or moving it to the check layer command class since I don't think it works for most commands.

When we add other check commands, we might want to make a BaseCheckCommand class to hold common flags like this.